src/pyrad_proc/pyrad/io/mxpol_config.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
```

Comment:
The main purpose is that I need to give the coordinates and altitudes of the MCH operational radars. For the MXPol I can read them from the files. 

Also, I have to convert the polarimetric variable names from our convention to the pyART convention (i.e. Zh to 'reflectivity'). I am not 100% certain how to proceed for this either.. as there is 'uncorrected' 'unfiltered' etc. What is your convention for this?
At the moment, we have a different (IDL) framework which converts the binary files to netCDF files and some corrections (i.e. signal to noise ratios) are already applied there (to be precise, it is not a correction, rather the bins with lower signal to noise ratios then a certain threshold are masked). Therefore, the Rhohv from MXPol is technically the 'corrected_cross_correlation_ratio'?